Formula/cppcheck.rb

```diff
@@ -33,6 +37,15 @@ def install
       system "make", "HAVE_RULES=no", "CFGDIR=#{prefix}/cfg"
     end
 
+    mkdir_p addons_dir # Create the directory or install will fail
+
+    # Patch the Makefile to install the python addons in the addons_dir folder,
+    # otherwise it would install them in the BIN folder
+    inreplace "Makefile" do |s|
+      s.sub! %r{install addons\/\*.py.*$}, "install addons/*.py #{addons_dir}"
+      s.sub! %r{install addons\/\*\/\*.py.*$}, "install addons/*/*.py #{addons_dir}"
+    end
```

Comment:
Is the add-on path baked into the scripts and the binary during installation? If not, it might be much easier to just move them out of `bin` after `make install` instead of patching the Makefile.